src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/EventPipeSessionConfiguration.cs

```diff
@@ -24,19 +24,22 @@ public sealed class EventPipeSessionConfiguration
         /// <param name="circularBufferSizeMB">The size of the runtime's buffer for collecting events in MB</param>
         /// <param name="requestRundown">If true, request rundown events from the runtime.</param>
         /// <param name="requestStackwalk">If true, record a stacktrace for every emitted event.</param>
+        /// <param name="rundownKeyword">If not null, override the default keyword for rundown events.</param>
         public EventPipeSessionConfiguration(
             IEnumerable<EventPipeProvider> providers,
             int circularBufferSizeMB = 256,
             bool requestRundown = true,
-            bool requestStackwalk = true) : this(circularBufferSizeMB, EventPipeSerializationFormat.NetTrace, providers, requestRundown, requestStackwalk)
+            bool requestStackwalk = true,
+            long? rundownKeyword = null) : this(circularBufferSizeMB, EventPipeSerializationFormat.NetTrace, providers, requestRundown, requestStackwalk, rundownKeyword)
```

Comment:
I'm not certain that adding the `rundownKeyword` parameter to the existing constructor is the correct design. As it is, it leads to an ambiguous understanding of how it interacts with the `requestRundown` parameter. For example, if a caller specifies `requestRundown = true, rundownKeyword = 0`, what does the caller expect to happen in the event pipe session? It seems that a separate constructor should be made so that the caller either chooses to collect the default rundown OR the caller specifies the rundown keyword, but not both. The rundown keyword parameter should be non-nullable.

Reply:
It is not, it broke binary compatibility, I missed this one when @noahfalk pointed that out before.